lib/util/splice.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * Plain old regular Array#splice may cause stackoverflow if we have too many items
+ */
```

Comment:
Can you add some more comments here about your findings? (also: maybe call this module `chunked-splice`; and could you use `//` comments?)